openvdb/openvdb/python/pyutil.h

```diff
@@ -7,10 +7,10 @@
 #include "openvdb/openvdb.h"
 #include "openvdb/points/PointDataGrid.h"
 #include <boost/python.hpp>
-#include <tbb/mutex.h>
-#include <map> // for std::pair
-#include <string>
+#include <mutex>
 #include <sstream>
+#include <string>
+#include <utility> // for std::pair
```

Comment:
This change from `<map>` to `<utility>` is a drive-by. I just noticed the `#include` didn't match the comment.